Improve website footer

kubami9 opened the pull request:
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

<!-- Ignore MarkdownBear -->
# _Do not_ delete any text other than where you are instructed.
<!-- Stop ignoring MarkdownBear -->

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

Added 'Projects' section to footer with the most popular FOSSASIA projects and 'More...' (link to FOSSASIA GitHub)

Preview link: https://kubami9.github.io/kubamichalski-GCIFOSSASIA/

kubami9 committed:
Add my blog article, lat & lng, and footer icon

Add my blog article
Add my lat and lng
Add Instagram FOSSASIA icon to the footer

kubami9 committed:
Add Russian language to the menu

Add Russian language option to the navigation

kubami9 committed:
Changed image to 240x240

Changed blog image to 240x240

kubami9 committed:
Improved website footer

Added Projects section in the footer with top FOSSASIA projects and button 'More...'

anshumanv commented:
Awesome changes @kubami. Can you see that older commits which are not indented in this PR are present, can you update your fork and include only commits which are relevant to this PR?

kubami9 commented:
@anshumanv sure! 👍

kubami9 committed:
Improve footer section

Improved website footer

Added Projects section in the footer with top FOSSASIA projects and button 'More...'

kubami9 committed:
Merge branch 'gh-pages' of https://github.com/kubami9/gci17.fossasia.org into gh-pages

kubami9 commented:
I'm very sorry for making a mess, I broke something while squashing and made unnecessarily commit which I'm not able to revert. Also there's a bug with some student image. I'll create second PR and close this.
